test/integration/main.test.js

```diff
@@ -13,18 +13,18 @@ describe('import', function() {
 let me;
 let ChatEngine;
 let ChatEngineYou;
-let globalChannel  = 'thisistheglobalchannel-whatever' + new Date().getTime();
+let globalChannel = 'thisistheglobalchannel-whatever' + new Date().getTime();
 
-describe('config', function() {
+describe('config', () => {
```

Comment:
Mocha discourages arrow function.
https://mochajs.org/#arrow-functions

Reply:
👍 we can revert them back.